Sources/NIO/ByteBuffer-core.swift

```diff
@@ -105,10 +105,6 @@ public struct ByteBufferAllocator {
                   hookedRealloc: @escaping @convention(c) (UnsafeMutableRawPointer?, size_t) -> UnsafeMutableRawPointer?,
                   hookedFree: @escaping @convention(c) (UnsafeMutableRawPointer?) -> Void,
                   hookedMemcpy: @escaping @convention(c) (UnsafeMutableRawPointer, UnsafeRawPointer, size_t) -> Void) {
-        #if !arch(arm) // only complain on 64-bit, this is unfortunate reality on 32-bit
```

Comment:
note to @Lukasa: This isn't necessary and covered much better by our allocation testing tests.